prisma/seed.ts

```diff
@@ -28,6 +29,9 @@ const seedUsers = async (n: number = NUMBER) => {
       const name = faker.name.fullName({ firstName, lastName });
 
       return {
+        role: faker.datatype.boolean()
+          ? UserRole.Requestor
+          : UserRole.Applicant,
```

Comment:
Not sure if we can really just randomize this here, as we will have seeded applicants and requestors which should have the right role and not just something randomized

Reply:
I just pushed a commit that makes it so the application and request seeders only select valid users ✨